eclass/cmake-utils.eclass

```diff
@@ -652,25 +652,57 @@ enable_cmake-utils_src_compile() {
 }
 
 _ninjaopts_from_makeopts() {
-	if [[ ${NINJAOPTS+set} == set ]]; then
```

Comment:
You are replacing this:
```
if [[ ${NINJAOPTS+set} == set ]]; then
    return 0
fi
```
with this:
```
[[ ${NINJAOPTS+set} == set ]] && \
    return 0
```
The latter is quite a bit uglier and wrapped onto 2 lines for no reason.